WebAPI/appsettings.json

```diff
@@ -51,6 +51,17 @@
     "RequireUniqueEmail": "true"
   },
 
+  "CalendarSettings" : {
+  	"StartWinterDay" : 28,
+  	"StartWinterMonth" : 9,
+  	"EndWinterDay" : 18,
+  	"EndWinterMonth" : 12,
+  	"StartSummerDay" : 22,
+  	"StartSummerMonth" : 2,
+  	"EndSummerDay" : 21,
+  	"EndSummerMonth" : 5
+  },
+
```

Comment:
Check line belowe (68) there are settings alreasdy - `StudyYear`, use these settings, don't create new one

Reply:
those are for the study year, not for the term